salt/states/service.py

```diff
@@ -349,6 +349,9 @@ def running(name,
     # Check if the service is available
     try:
         if not _available(name, ret):
+            if 'test' in __opts__ and __opts__['test']:
```

Comment:
Ah, that seems better, thanks; there's probably the same thing to be said in #40696 i guess?